cluster-stamp.json

```diff
@@ -838,13 +839,17 @@
                             "1",
                             "2",
                             "3"
-                        ]
+                        ],
+                        "upgradeSettings": {
+                            "maxSurge": "33%"
```

Comment:
It's possible.  33% is the recommended value, it's just now configurable.  At our current node counts, this will basically continue to be 1 node at a time.  It would change if we had higher node counts though in a node pool.  I don't know this chart well enough to know how to express that.

Reply:
if you agree, I'd just bump that number from `1` to `2`. The reason behind this is that I'm guessing that we might have two nodes being upgraded at the same time since we have 2 nodepools.

Reply:
Sure.  Does that have a ripple effect on the other numbers to the right of it?  Again, sorry, I'm not sure up to speed on that chart.

Reply:
It shouldn't be the case because we were suggested to go with `/16`. I mean, there is already room for it.

Reply:
Sounds good, updated.  I just wanted to make sure some values to the right of it were not a fn() of this value somehow, didn't want the chart not to "sum up" properly in some way.